content/influxdb/v2.0/reference/cli/influx/v1/dbrp/create.md

```diff
@@ -0,0 +1,36 @@
+---
+title: influx v1 dbrp create
```

Comment:
let's reuse the example from here: https://docs.influxdata.com/influxdb/v2.0/query-data/influxql/#map-unmapped-buckets

to show a fully formed usage example?  When using the CLI profiles, the majority of the parameters will be provided (URL, org, token)...which should simplify things!

(it should be noted that the example on that page includes, erroneously, both the org and org_id.  I'd suggest simplifying that by picking one or the other.  The description says that you can use either.